activerecord/test/cases/schema_dumper_test.rb

```diff
@@ -433,6 +433,28 @@ def test_schema_dump_include_limit_for_float4_field
       output = dump_table_schema "numeric_data"
       assert_match %r{t\.float\s+"temperature_with_limit",\s+limit: 24$}, output
     end
+
+    def test_schema_dump_keeps_enum_intact_if_it_contains_comma
+      original, $stdout = $stdout, StringIO.new
+
+      migration = Class.new(ActiveRecord::Migration::Current) do
+        def up
+          create_enum "enum_with_comma", ["value1", "value,2", "value3"]
+        end
+
+        def down
+          drop_enum "enum_with_comma"
+        end
+      end
+
+      migration.migrate(:up)
+      output = dump_all_table_schema
+
+      assert_match %r{create_enum "enum_with_comma", \["value1", "value,2", "value3"\]}, output
+    ensure
+      migration.migrate(:down) if migration
```

Comment:
Already suggested in the previous snippet - without regexp looks simpler and `migration` can't be nil
```suggestion
      assert_includes output, 'create_enum "enum_with_comma", ["value1", "value,2", "value3"]', output
    ensure
      migration.migrate(:down)
```